polygeist/test/polygeist-opt/reachingDefinitions.mlir

```diff
@@ -454,3 +454,14 @@ func.func @caller() {
   %0 = func.call @callee(%ptr) : (memref<i32>) -> memref<i32>
   return
 }
+
+// -----
+
+// COM: Test that the analysis does not break with non-functions
+
```

Comment:
Can you add some CHECKS to test that the mods/pmods are what you expect. Is that doable?

Reply:
Done!